### ISSUE TITLE
How to use optimize_image.py

### ISSUE THREAD

**Issue by idealboy** [yosinski/deep-visualization-toolbox#117]
Dear Sir,

How can I use the script optimize_image.py to process my Images? Thank you!

**Comment by pingjunLi** [comment]
Dear Sir,
Have you solved your problem? I have the same problem, please

**Comment by heitorrapela** [comment]
Hey there, checkout this link, maybe it could help you.
[Running Optimize Image](https://github.com/yosinski/deep-visualization-toolbox/blob/master/doc/running-optimize-image.md)

**Comment by pingjunLi** [comment]
yes,I known that,but when I run the command like './optimize_image.py --decay 0      --blur-radius 0.5 --blur-every 4  --small-norm-percentile 50     --max-iter 500  --lr-policy progress --lr-params "{'max_lr': 100.0, 'desired_prog': 2.0}",it always turn out the valueError : Mean shape incompatible with input shape. It makes me confused.

**Comment by heitorrapela** [comment]
I'm running the deep-visualization-toolbox with googLeNet. I had this error too, but I correct it. Check this [StackOverflow](https://stackoverflow.com/questions/30808735/error-when-using-classify-in-caffe): You need to caffe/python/caffe/io.py and replace some lines.
